tools/ci.psm1

```diff
@@ -123,14 +123,6 @@ function Invoke-CIInstall
     # Make sure we have all the tags
     Sync-PSTags -AddRemoteIfMissing
 
-    if(Test-DailyBuild)
-    {
-        if ($env:BUILD_REASON -eq 'Schedule')
```

Comment:
This one line can be removed now.

But otherwise, this changes how we name the builds for daily builds if you want to add a comment.